### DISCUSSION TITLE
بعد نصب لینک میاره ولی وارد پانل نمیشه

### DISCUSSION THREAD

**Discussion by behcom** [hiddify/Hiddify-Manager#915] [Q&A سوالات رایج]
نصب میشه همه لینک ها رو هم میاره ولی هرکدوم از لینک ها رو کپی میکنم باز نمیکنه وارد پنل بشه

**Comment by behcom** [comment]
روی دو تا ورژن لینوکس هم نصب کردم لینک میاره ولی وارد  پنل نمیشه

**Comment by saeidghodrati** [comment]
من هم همین مشکل دارم در لینک ادمین صفحه سفید نمایش داده میشه
